### DISCUSSION TITLE
Ownership Transfer

### DISCUSSION THREAD

**Discussion by trs** [QuorumDMS/ftp-srv#207] [General]
Greetings!
You may have noticed a lack of activity in this repository lately. There hasn’t been much free time for me to work on this.
Autovance, the company I work for, has graciously allowed me to move this under their org to sponsor development! What a great opportunity.

### What does this mean?
Essentially work can be done on this repository during work hours, meaning more effort and time can be put in. For you, the end user/contributor, not much will change about the current version being used.

### What does the future hold?
The current version will be placed into maintenance mode, while a brand new re-write will be done. This will make it easier to work with and, with everything that has been learned, reduce the bugs!

Thanks for sticking with us through this transition period!

**Comment by JonathanRowell** [comment]
Fine, I'll look forward to it.

**Comment by quiquelhappy** [comment]
Hello! Any new updates on this? I'm a bit worried by this https://www.npmjs.com/advisories/1445! Do you have an ETA? Also, can anyone recommend an alternative package meanwhile?

**Comment by matt-forster** [comment]
@quiquelhappy We do not have an update for the new version as of right now. That advisory was never brought to our attention, and we will look at correcting it ASAP.

**Comment by matt-forster** [comment]
Thanks @quiquelhappy for bringing this to our attention - we've published a security advisory and a patch. It looks like NPM had emailed us, but as it was the 23rd of December, it fell through the cracks, and was never followed up on. No excuse, but we will be publishing back ported fixes for versions all the way through to v1 this afternoon.

https://github.com/autovance/ftp-srv/security/advisories/GHSA-jw37-5gqr-cf9j

e449e75219d918c400dec65b4b0759f60476abca
